apis/Google.Cloud.Talent.V4Beta1/docs/history.md

```diff
@@ -2,6 +2,14 @@
 
 (This API has changed significantly over time, and the history is tricky to backfill. Future changes will be tracked more closely.)
```

Comment:
nit: Can this be removed now? I think it was added when the history.md file was first generated. Or maybe just add this same comment or a similar one to the end of the file saying "Previous changes are tricky to backfill..."

Reply:
Yup, will have a look now.